cmake/scripts/android/Install.cmake

```diff
@@ -156,31 +147,140 @@ add_bundle_file(${ASS_LIBRARY} ${libdir} "")
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/res/drawable/splash.jpg COPYONLY)
+configure_file(${CMAKE_SOURCE_DIR}/media/splash.jpg
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/res/drawable-xxxhdpi/splash.jpg COPYONLY)
+configure_file(${CMAKE_SOURCE_DIR}/media/icon80x80.png
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/res/drawable/ic_recommendation_80dp.png COPYONLY)
+
+# other android packaging
+configure_file(${ANDROID_PACKAGING_SOURCE_DIR}/xbmc/xbmc.properties.in
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/res/raw/xbmc.properties @ONLY)
+
+configure_file(${ANDROID_PACKAGING_SOURCE_DIR}/xbmc/AndroidManifest.xml.in
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/AndroidManifest.xml @ONLY)
+
+configure_file(${ANDROID_PACKAGING_SOURCE_DIR}/xbmc/build.gradle.in
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/build.gradle @ONLY)
+
+configure_file(${ANDROID_PACKAGING_SOURCE_DIR}/xbmc/strings.xml.in
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/res/values/strings.xml @ONLY)
+
+configure_file(${ANDROID_PACKAGING_SOURCE_DIR}/xbmc/colors.xml.in
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/res/values/colors.xml @ONLY)
+
+configure_file(${ANDROID_PACKAGING_SOURCE_DIR}/xbmc/searchable.xml.in
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/res/xml/searchable.xml @ONLY)
+
+configure_file(${TOOLCHAIN}/sysroot/usr/lib/${HOST}/libc++_shared.so
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/lib/${CPU}/libc++_shared.so COPYONLY)
+
+configure_file(${NDKROOT}/prebuilt/android-${NDK_CPU}/gdbserver/gdbserver
+               ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/lib/${CPU}/gdbserver COPYONLY)
+
+# todo: gdb setup?
+
+string(TOLOWER "${CMAKE_BUILD_TYPE}" CMAKE_BUILD_TYPE_LC)
+
+add_custom_command(OUTPUT ${ANDROID_PACKAGING_BINARY_DIR}/.copy-libs-done
+                   COMMAND find ${CMAKE_INSTALL_PREFIX}/ -name \"*.so\" -exec cp -fp {} ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/lib/${CPU}/ \\\;
+                   COMMAND find ${DEPENDS_PATH}/lib/${APP_NAME_LC}/addons -name \"*.so\" -exec cp -fp {} ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/lib/${CPU}/ \\\;
+                   COMMAND ${CMAKE_COMMAND} -E touch ${ANDROID_PACKAGING_BINARY_DIR}/.copy-libs-done
+                   DEPENDS bundle
+                   COMMENT "Copying libs for packaging")
+
+# hack: I don't like this step and I'm not sure why it's needed however gradle doesn't include
+# files in the libs/ directory that don't start with lib (maybe a gradle bug?)
+add_custom_command(OUTPUT ${ANDROID_PACKAGING_BINARY_DIR}/.rename-libs-done
+                   COMMAND ls *.so | grep -v "^lib" | xargs -I {} mv {} lib{}
+                   COMMAND ${CMAKE_COMMAND} -E touch ${ANDROID_PACKAGING_BINARY_DIR}/.rename-libs-done
+                   WORKING_DIRECTORY ${ANDROID_PACKAGING_BINARY_DIR}/xbmc/lib/${CPU}/
+                   DEPENDS ${ANDROID_PACKAGING_BINARY_DIR}/.copy-libs-done
+                   COMMENT "Renaming libs for packaging")
```

Comment:
As odd as it seems, its actually a requirement. Theres workarounds and other things to not have to have the lib prefix, but its really not worth the hassle from what ive seen